eras/shelley-ma/impl/src/Cardano/Ledger/ShelleyMA/TxWits.hs

```diff
@@ -1,39 +1,5 @@
-{-# LANGUAGE DataKinds #-}
-{-# LANGUAGE FlexibleContexts #-}
-{-# LANGUAGE FlexibleInstances #-}
-{-# LANGUAGE MultiParamTypeClasses #-}
-{-# LANGUAGE ScopedTypeVariables #-}
-{-# LANGUAGE TypeApplications #-}
-{-# LANGUAGE TypeFamilies #-}
-{-# LANGUAGE UndecidableInstances #-}
-{-# OPTIONS_GHC -Wno-orphans #-}
-
-module Cardano.Ledger.ShelleyMA.TxWits () where
-
-import Cardano.Ledger.Core (EraTxWits (..))
-import Cardano.Ledger.Crypto (StandardCrypto)
-import Cardano.Ledger.Shelley.TxWits
-  ( ShelleyTxWits,
-    addrShelleyTxWitsL,
-    bootAddrShelleyTxWitsL,
-    scriptShelleyTxWitsL,
+module Cardano.Ledger.ShelleyMA.TxWits
+  {-# DEPRECATED "Should not have been exposed in the first place" #-}
```

Comment:
It shouldn't have been exposed?  We're still exposing Mary.TxWits in the new implementation:  https://github.com/input-output-hk/cardano-ledger/pull/3175/files#diff-3c54618278f3efba674af5e7fc84e6fe414da600e08b9ddc3603868ab5603ed5R11

Reply:
Actually you are almost right, we are exposing it, but in Allegra, not in Mary. I'll fix that. See:

https://github.com/input-output-hk/cardano-ledger/blob/01f523a72c132259466f32c69a4a88d4b555f96d/eras/mary/impl/cardano-ledger-mary.cabal#L47-L53

Reply:
Fixed

Reply:
FYI. It shouldn't have been exposed, because it does not export anything. The only thing that it exports is type class instance for EraTxWits, which is unusable unless you have the concrete era type (eg. `MaryEra`) in scope, which itself is exported together with the instance. So `TxWits` modules in Allegra and Mary are suppose to be hidden

Reply:
Makes sense! Thanks for the explanation